### PR TITLE
111223

### PR DESCRIPTION
**Pull request by George-ast**
homework

### CODE REVIEW

File: home3.3.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+"""old_text."""
```
Comment (antohaUa) [inline home3.3.py]: Please add task description into the docstring. Plus combine both files to have one file with full required solution

**Review by antohaUa — CHANGES_REQUESTED** [review]
Required docstring with task description and single file for solution.
All other good

File: home3.3.py (hunk 1 above)
Comment (George-ast) [reply home3.3.py]: done

File: home3.3.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+we have norway text in old style formatting
+re-write the same text as:
+1 string with format() call
+2 f-string
+use linter(https://github.com/wemake-services/wemake-python-styleguide)
+to check your new created python module for possible linter errors
+try to run code from pycharm/command line
+
+norway_text = 'Automatisering akselererer %syeblikket da roboter vil erobre planeten v%sr. (%s)' % ('ø', 'å', 'Æ')
```
Comment (antohaUa) [inline home3.3.py:11]: These both lines should be out of docstring text.
This is code ( aka initial data)

File: home3.3.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+print(norway_text).
+
+"""
+
+symbol1 = 'ø'
+symbol2 = 'å'
+symbol3 = 'Æ'
+
+t2 = ('Automatisering akselererer, {0}syeblikket da roboter'
+    'vil erobre planeten v{1} ({2})').format(symbol1, symbol2, symbol3)
```
Comment (antohaUa) [inline home3.3.py:21]: E128 continuation line under-indented for visual indent
Pay attention to indents too
```suggestion
t2 = ('Automatisering akselererer, {0}syeblikket da roboter'
      'vil erobre planeten v{1} ({2})').format(symbol1, symbol2, symbol3)
```

**Review by antohaUa — APPROVED** [review]
Good

**Pull request merged**